### PR TITLE
build: Integrate Happo for visual diffs

### PR DESCRIPTION
**Pull request by milesj**
to: @milesj @stefhatcher @hayes @alecklandgraf

## Description

Integrates Happo for visual diffs within a PR.

## Motivation and Context

Catch visual regressions (which happen too often).

## Testing

Ran it locally (below).

## Screenshots

https://happo.io/a/177/p/205/report/dev-e7ef4d3b8074fe7c0563

## Checklist

- My code follows the style guide of this project.
- I have updated or added documentation accordingly.
- I have read the CONTRIBUTING document.

### CODE REVIEW

File: packages/core/src/components/DateTime.story.tsx [hunk 1]
```diff
@@ -5,15 +5,17 @@ import DateTime from './DateTime';
 const future = new Date();
 future.setDate(future.getDate() + 12);
 
+const fixedDate = new Date(2019, 1, 1, 0, 0, 0);
```
Comment (stefhatcher) [inline packages/core/src/components/DateTime.story.tsx]: DateTimeRange will need fixed dates too

Comment (stefhatcher) [reply packages/core/src/components/DateTime.story.tsx]: Maybe DatePicker too? and DatePickerInput?

**Comment by stefhatcher** [comment]
is it possible to have happen generate hover states? looking at the snapshots for tooltip, not seeing any actual examples of tooltip. maybe add some stories for it with the tooltip rendered?

**Comment by milesj** [comment]
@stefhatcher Cannot do hover states. I'll disable tooltip for now since it can't be rendered without a hover.

**Comment by stefhatcher** [comment]
might need to disable Dropdown, MenuToggle, Modal, Overlay, Portal, Sheet

**Comment by stefhatcher** [comment]
DOOOOOOPE

**Pull request merged**